core/runtime/src/main/java/org/eclipse/sw360/antenna/workflow/stubs/AbstractComplianceChecker.java

```diff
@@ -36,28 +37,34 @@ public AbstractComplianceChecker() {
         this.workflowStepOrder = VALIDATOR_BASE_ORDER;
     }
 
-    public abstract IPolicyEvaluation evaluate(Collection<Artifact> artifacts) throws AntennaException;
+    public abstract IPolicyEvaluation evaluate(Collection<Artifact> artifacts);
 
     public abstract String getRulesetDescription();
 
     @Override
-    public Collection<Artifact> process(Collection<Artifact> artifacts) throws AntennaException {
+    public Collection<Artifact> process(Collection<Artifact> artifacts) throws ExecutionException {
         LOGGER.info("Evaluate compliance rule set: {}", getRulesetDescription());
         IPolicyEvaluation evaluation = evaluate(artifacts);
         LOGGER.info("Rule evaluation done");
         LOGGER.info("Check evaluation results...");
-        execute(evaluation);
+        try {
+            execute(evaluation);
+        } catch (FailCausingException e) {
+            // ToDo: A FailCausingException should not end the build but should be reflected in the workflow result
```

Comment:
TODO

Reply:
to be grep safe and to support idea highlighting:
```suggestion
            // TODO: A FailCausingException should not end the build but should be reflected in the workflow result
```

Reply:
Done